plugins/python_3_script/Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN apt-get update && \
 ENV PYTHONUSERBASE=/var/cache/python_dependencies \
 		PYTHONPATH=/var/cache/python_dependencies
 
+RUN mkdir $PYTHONUSERBASE && chmod 777 $PYTHONUSERBASE
```

Comment:
Is there a more restrictive permission we can give?

Reply:
Actually, when creating that folder we are at the root level, so in order to allow `nobody` to read/write for that it's required to `chmod 777`. The case `755` won't work, because the folder owner is not `nobody` 😃

Reply:
Could we try something like this? 

setfacl -m -R u:nobody:rwx $PYTHONUSERBASE

Reply:
It does not work, found the other way to do so
```
RUN mkdir $PYTHONUSERBASE && chown nobody: $PYTHONUSERBASE && chmod u+w $PYTHONUSERBASE
```